src/task/style.css

```diff
@@ -104,6 +104,8 @@ body {
   font-size: medium;
 }
 
+#category,
+#level,
 #status {
```

Comment:
Isn't it better to have a class for it?

Reply:
The class is already there and it is applied to many more elements.
That class is used in JS for a separate purpose
The code needs refactoring and I guess @Pavangbhat is already taking care of it separately